doc-surrealdb_versioned_docs/version-1.x/reference-guide/full-text-search.mdx

```diff
@@ -6,98 +6,42 @@ description: In SurrealDB, Full-Text Search supports advanced features like basi
 
-Full-Text search is a feature that allows search capabilities within a database. This feature can be customized with specific analyzers and tokenizers to refine the search process.
-
-In SurrealDB, Full-Text Search supports advanced features like basic and advanced text matching, proximity searches, result ranking, and keyword highlighting⁠.
-
-Full-Text Search in SurrealDB is [ACID-compliant](https://en.wikipedia.org/wiki/ACID), which ensures data integrity and reliability.⁠
-
-## Core Concepts and Definitions
-
-In order to perform Full-Text Search on textual data within SurrealDB, we leverage analyzers, tokenizers, indexes and filters to optimize the search precision.
-We also use different search functions to highlight keywords and improve their relevance.
+## Define tables and fields
+To implement Full-Text Search it is important that your data is first defined using SurrealDB [tables](/docs/surrealdb/surrealql/statements/define/table) and [fields](/docs/surrealdb/surrealql/statements/define/field). 
 
 ## Analyzers
-
-[Analyzers](/docs/surrealdb/surrealql/statements/define/analyzer) are configurations that use tokenizers or filters to prepare text data for searching.
-
-They are foundational to how text is processed in Full-Text Search. It is defined by its name, a set of tokenizers, and a collection of filters.
-
-## Tokenizers
-
-[Tokenizers](/docs/surrealdb/surrealql/statements/define/analyzer#tokenizers) break down text into manageable tokens based on specified rules like spaces or punctuation. 
-If we tokenize the sentence '*Getting started with SurrealDB*' based on space, it would be broken down into tokens: Getting, started, with, SurrealDB.
-
-```surql
--- Defining a tokenizer that splits text into words based on spaces
-DEFINE ANALYZER space_tokenizer TOKENIZERS blank;
-
-```
-
-## Filters
-
-[Filters](/docs/surrealdb/surrealql/statements/define/analyzer#filters) process tokens for further refinement, such as converting to lowercase, removing special characters or breaking down tokens into useful prefixes to prepare for more effective searching.
+Once the data is in your tables, you can use customized [analyzers](/docs/surrealdb/surrealql/statements/define/analyzer) to define rules for how your textual data should be searched.
+An analyzer includes [tokenizers](/docs/surrealdb/surrealql/statements/define/analyzer#tokenizers) and [filters](/docs/surrealdb/surrealql/statements/define/analyzer#filters) which help break down text into manageable tokens and refine the search.
 
 ```surql
 -- Combining tokenizers and filters into a custom analyzer
 DEFINE ANALYZER custom_analyzer TOKENIZERS blank FILTERS lowercase, snowball(english);
-
 ```
-
 ## Define a Full-Text Index
+To make a text field searchable, you need to set up a [full-text index](/docs/surrealdb/surrealql/statements/define/indexes#full-text-search-index) on it by using the 'search' keyword.
```

Comment:
```suggestion
To set up a Full text search, you'll need to first define a [full-text index](/docs/surrealdb/surrealql/statements/define/indexes#full-text-search-index) on your field or column and then call your analyzer on that index using the `SEARCH ANALYZER` clause at the end of your index.
```

Reply:
Hi @Ekwuno, can you please clarify what you mean by the `SEARCH ANALYSER` clause in SurrealDB? We have the `SEARCH` keyword which tells the parser what type of index it is. And the ANALYZER keyword which is followed by the name of the custom analyser. I am not sure if `clause` is the right terminology to use.